qt_ui/widgets/simspeedcontrols.py

```diff
@@ -33,3 +33,7 @@ def speed_changed(self, speed_factor: int) -> None:
 
     def on_sim_speed_reset(self, speed_setting: SimSpeedSetting) -> None:
         self.buttons[speed_setting].setChecked(True)
+
+    def setEnabled(self, enabled: bool) -> None:
```

Comment:
If you override this you need to call the base method, and you also need to override `setDisabled`.